modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -302,6 +302,14 @@
 		],
 		"to": "serveImage(PortalUtil.getHttpServletRequest(param#0#), PortalUtil.getHttpServletResponse(param#1#))"
 	},
+	{
+		"classNames": [
+			"AssetCategoryLocalService",
+			"AssetCategoryLocalServiceUtil"
```

Comment:
@MicaelleSilvaa, can we also include here the `AssetCategoryService` and `AssetCategoryServiceUtil`?

If these classes exist in `master` and also have the methods mentioned in `from` and `to` then it makes sense we include them. From now on, if it's applicable, we should always use, for these specific cases, the following classes:

- `*LocalService`
- `*LocalServiceUtil`
- `*Service`
- `*ServiceUtil`

Reply:
@MicaelleSilvaa, do you have any update about this comment?

Thanks